tests/check_data.py

```diff
@@ -0,0 +1,30 @@
+import pytest
```

Comment:
Because we'll want to use this from the notebook itself, it's part of the library, rather than part of the test suite. Let's move it out of the `tests` dir and into the main library dir with the other `.py` files (e.g. `sumstats.py`).

Reply:
Also - we're not using `pytest` here, can remove.